### ISSUE TITLE
Stormlib / MPQEditor crash when extracting patched file

### ISSUE THREAD

**Issue by jleclanche** [ladislav-zezula/StormLib#59]
When extracting the files inside this patched MPQ, stormlib crashes:

```
adys@azura ~/HSB % sha256sum 3388.direct/base-Win.MPQ 3388.direct/Updates/hs-0-3604-Win-final.MPQ
3e194fd8b4642887c8ef043bcbe220f6375a3a1f5720bcb38fa9be6c77308ff9  3388.direct/base-Win.MPQ
bb24c23cda87f6bedd2fda22b37378decbcb78983f41aa2d3028ffcce47d2561  3388.direct/Updates/hs-0-3604-Win-final.MPQ
```

I think the patch chain is wrong (I can't figure out on top of what build that patch mpq is supposed to be), but stormlib shouldnt crash.

**Comment by Krakean** [comment]
Upload somewhere this mpq.

**Comment by ladislav-zezula** [comment]
Fixed in commit f608c0798420fd0d7a472a831ecef00bbdf4c7f0.
